packages/react-tabs/src/next/index.ts

```diff
@@ -1,2 +1,3 @@
 export * from '../index';
 export { Pivot } from './Pivot';
+export * from './Tabs';
```

Comment:
let's not export from `index` yet until we are done will all the breaking changes and ready to deprecate `Pivot` in favor of `Tabs`